### ISSUE TITLE
Compile List of assosiated projects.

### ISSUE THREAD

**Issue by IGBC** [Rust-Retro/RWG#2]
Exactly what it says on the tin. As of right now prior art relating to retro platorms exists, out in the ether. We should probably complile a list of these projects in one place for future reference as the RWG works to grow support.

**Comment by cr1901** [comment]
https://github.com/cr1901/parasol Parasol will be an SSL accelerator for old IBM PC compatibles to make them fast enough to do SSL under their own power. I have not worked out all the hardware details yet, but the firmware will be written in Rust.

**Comment by jonas-schievink** [comment]
* [cargo-n64](https://github.com/rust-console/cargo-n64) – a Cargo subcommand that helps with building Nintendo 64 games
* [prussia](https://github.com/ZirconiumX/prussia) – a PS2 SDK
* [Monotron](https://github.com/thejpster/monotron) and [Neotron](https://github.com/Neotron-Compute) should probably get an honorary mention too

**Comment by IGBC** [comment]
>  * [Monotron](https://github.com/thejpster/monotron) and [Neotron](https://github.com/Neotron-Compute) should probably get an honorary mention too

Yes I should probably finish the Neotron's PCB at some point * cough *

**Comment by IGBC** [comment]
Yet another rust to C compiler: 

https://github.com/thepowersgang/mrustc

**Comment by thejpster** [comment]
Rust on the GBA: https://github.com/rust-console/gba
Rust on the PlayStation 2: https://github.com/rust-console/ps2-bios-book

**Comment by thejpster** [comment]
Bare-metal Rust OS for x86-64: https://os.phil-opp.com/ (could be adapted for i386 relatively easily ...?)

**Comment by thejpster** [comment]
Jason Turner's x86 to 6502 transpiler: https://github.com/lefticus/x86-to-6502

**Comment by sajattack** [comment]
https://github.com/xTibor/rust-on-c64

**Comment by IGBC** [comment]
Theres a pr for tbis now #4 

Add your suggestions by ammending the pr

**Comment by IGBC** [comment]
is this included anywhere: https://github.com/autc04/Retro68/blob/master/README.md ?
